broker-daemon/proto/broker.proto

```diff
@@ -222,7 +244,7 @@ service AdminService {
    * └───────────────────┴───────────────────┘
    * ```
    */
-  rpc HealthCheck (google.protobuf.Empty) returns (HealthCheckResponse) {
+  rpc HealthCheck (HealthCheckRequest) returns (HealthCheckResponse) {
```

Comment:
@treygriffith did we want to keep the flag hidden or provide an example in the proto?

Reply:
No, I think adding it to the docs would be more confusing - I'm fine with it being a hidden field for developer usage.